packages/console/src/Console.tsx

```diff
@@ -391,52 +504,61 @@ export class Console extends PureComponent {
       const objectHistoryMap = new Map(state.objectHistoryMap);
       const objectMap = new Map(state.objectMap);
 
-      const disableOldObject = (object, isRemoved = false) => {
-        const { name } = object;
-        const oldIndex = objectHistoryMap.get(name);
+      const disableOldObject = (
+        object: VariableDefinition,
+        isRemoved = false
+      ) => {
+        const { title } = object;
+        assertNotNull(title);
+        const oldIndex = objectHistoryMap.get(title);
         // oldIndex can be -1 if a object is active but doesn't have a command in consoleHistory
         // this can happen after clearing the console using 'clear' or 'cls' command
+        assertNotNull(oldIndex);
```

Comment:
Instead of assertNotNull here, just check if `oldIndex != null && oldIndex >= 0` in the if statement below. We may at some point truncate the console history, in which case the objectHistoryMap may return null.